index.html

```diff
@@ -0,0 +1,15 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <link rel="icon" href="/favicon.ico" />
```

Comment:
where is the favicon file?

Reply:
there's is no favicon, i forgot to delete this code

Reply:
Don’t delete the code. But provide the icon. An icon is essential for an app